app/models/item.rb

```diff
@@ -4,7 +4,7 @@ class Item < ApplicationRecord
 
   validates :name, presence: true
   validates :max_reservation_days, numericality: { greater_than_or_equal_to: 0, less_than_or_equal_to: 365 }
-  validates :max_borrowing_days, numericality: { greater_than_or_equal_to: 0 }
+  validates :max_borrowing_days, numericality: { greater_than: 0 }
```

Comment:
This makes sense.

To keep track of these "on-the-side" changes, it might be nice to give a summary of changed things (that are not obvious from the PR title) in the PR.